.github/workflows/push-to-master.yml

```diff
@@ -1,79 +1,6 @@
 name: push-to-master
-on:
-  push:
-    branches:
-      - master
+on: pull_request
 jobs:
-  test-ray-master:
-    runs-on: ubuntu-latest
```

Comment:
why change this?

Reply:
For test purpose; this should be reverted before merge.